### PR TITLE
Fix delete responses return type and update document comments

### PR DESCRIPTION
**Pull request by kapilraajP**
Fixes wso2-enterprise/choreo#4473
Fixes wso2-enterprise/choreo#4664

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: endpoint.bal [hunk 1]
```diff
@@ -105,10 +108,10 @@ public client class Client {
...
-    # + messageGroupId -  Message group which a message belongs. only to FIFO queues
-    # + messageDeduplicationId - Message deduplicationId Id. only to FIFO queues
-    # + delaySeconds - Length of time for which to delay a specific message. On FIFO queue can't set for a message
+    # + messageAttributes - (optional) Message attributes for sending a message 
+    # + messageGroupId - (optional) Message group which a message belongs. only applicable to FIFO queues
+    # + messageDeduplicationId - (optional) Message deduplicationId Id. only applicable to FIFO queues
+    # + delaySeconds - (optional) Length of time for which to delay a specific message. On FIFO queue can't set for a message
     # + return - If success, details of the sent message, else returns error
     @display {label: "Send message in queue"}
     remote isolated function sendMessage(@display {label: "Message body to send"} string messageBody,
```
Comment (abeykoon) [inline endpoint.bal:117]: `Message Body` is enough is now user is in the context of `send`

File: endpoint.bal [hunk 2]
```diff
@@ -105,10 +108,10 @@ public client class Client {
...
-    # + messageAttributes - Non-mandatory message attributes for sending a message 
-    # + messageGroupId -  Message group which a message belongs. only to FIFO queues
-    # + messageDeduplicationId - Message deduplicationId Id. only to FIFO queues
-    # + delaySeconds - Length of time for which to delay a specific message. On FIFO queue can't set for a message
+    # + messageAttributes - (optional) Message attributes for sending a message 
+    # + messageGroupId - (optional) Message group which a message belongs. only applicable to FIFO queues
+    # + messageDeduplicationId - (optional) Message deduplicationId Id. only applicable to FIFO queues
+    # + delaySeconds - (optional) Length of time for which to delay a specific message. On FIFO queue can't set for a message
     # + return - If success, details of the sent message, else returns error
     @display {label: "Send message in queue"}
```
Comment (abeykoon) [inline endpoint.bal:116]: Shall we call it `Send Message`?

File: endpoint.bal [hunk 3]
```diff
@@ -232,11 +235,11 @@ public client class Client {
     #
     # + queueResourcePath - Resource path to the queue from the host address. e.g.: /610968236798/myQueue.fifo
     # + receiptHandle - Receipt Handle parameter for the message(s) to be deleted
-    # + return - Whether the message(s) were successfully deleted or whether an error occurred
+    # + return - Null when the message(s) were successfully deleted or whether an error occurred
     @display {label: "Delete message in queue"}
     remote isolated function deleteMessage(@display {label: "Resource path to queue"} string queueResourcePath,
```
Comment (abeykoon) [inline endpoint.bal:240]: `Queue Resource Path`

File: endpoint.bal (hunk 2 above)
Comment (kapilraajP) [reply endpoint.bal]: resolved

**Pull request closed without merging**